src/render.js

```diff
@@ -36,13 +36,17 @@ export default function render<T>(
   const update = updateWithAct(renderer, wrap);
   const instance = renderer.root;
 
+  for (let query in queries) {
+    queries[query] = queries[query].bind(null, instance);
+  }
+
   addToCleanupQueue(renderer.unmount);
 
   return {
-    ...getByAPI(instance),
-    ...queryByAPI(instance),
-    ...findByAPI(instance),
-    ...a11yAPI(instance),
+    ...(queries: {
+      [key: $Keys<typeof queries>]: (...rest: Array<any>) => any,
```

Comment:
Struggling a bit on how to type it better, so that custom queries are inferred without forced generics. We can improve it later though